docs/environment-variables.md

```diff
@@ -92,8 +92,6 @@
 
 | Name                                    | Description                                                                  | Default |
 | :-------------------------------------- | :--------------------------------------------------------------------------- | :------ |
-| FEATURE_FLAG_YOUTH_RISK_ASSESSMENT      | Set to `true` to enabled youth risk assessment for Secure Training Centres and Secure Childrens Homes |         |
```

Comment:
Reminder: Once this is deployed to master, we'll need a chore to go through and cleanup the envvars on all the environments